 
 -------------- celery@Evans-MacBook-Air.local v5.3.4 (emerald-rush)
--- ***** ----- 
-- ******* ---- macOS-13.2.1-arm64-arm-64bit 2025-07-15 23:48:09
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         fintellic:0x102968c50
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 4 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . app.tasks.filing_tasks.process_filing_task
  . app.tasks.filing_tasks.process_pending_filings
  . app.tasks.filing_tasks.send_filing_notifications

[2025-07-15 23:48:09,631: WARNING/MainProcess] /Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
whether broker connection retries are made during startup in Celery 6.0 and above.
If you wish to retain the existing behavior for retrying connections on startup,
you should set broker_connection_retry_on_startup to True.
  warnings.warn(

[2025-07-15 23:48:09,640: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-07-15 23:48:09,641: WARNING/MainProcess] /Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
whether broker connection retries are made during startup in Celery 6.0 and above.
If you wish to retain the existing behavior for retrying connections on startup,
you should set broker_connection_retry_on_startup to True.
  warnings.warn(

[2025-07-15 23:48:09,643: INFO/MainProcess] mingle: searching for neighbors
[2025-07-15 23:48:10,650: INFO/MainProcess] mingle: all alone
[2025-07-15 23:48:10,671: INFO/MainProcess] celery@Evans-MacBook-Air.local ready.
[2025-07-16 07:13:29,781: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[e35e2c13-5e49-482c-a17f-cb204c1e5f50] received
[2025-07-16 07:13:29,797: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[4282e026-a0c0-4a96-9247-f217f063115f] received
[2025-07-16 07:13:29,825: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[b3e7cd29-9a11-4e2f-bc63-d81ffcbe9273] received
[2025-07-16 07:13:29,848: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[eb981ace-9c30-47dc-af98-88cb8d87be5b] received
[2025-07-16 07:13:29,884: INFO/ForkPoolWorker-4] Starting processing for filing 67
[2025-07-16 07:13:29,884: INFO/ForkPoolWorker-2] Starting processing for filing 66
[2025-07-16 07:13:30,272: ERROR/MainProcess] Process 'ForkPoolWorker-2' pid:73903 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 07:13:30,302: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 0.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 0.
"""
[2025-07-16 07:13:30,429: ERROR/MainProcess] Process 'ForkPoolWorker-4' pid:73905 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 07:13:30,443: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 1.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 1.
"""
[2025-07-16 07:13:35,808: INFO/ForkPoolWorker-6] Starting processing for filing 68
[2025-07-16 07:13:35,912: ERROR/MainProcess] Process 'ForkPoolWorker-6' pid:75943 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 07:13:35,926: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 2.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 2.
"""
[2025-07-16 07:15:16,742: INFO/ForkPoolWorker-7] Starting processing for filing 69
[2025-07-16 07:15:16,874: ERROR/MainProcess] Process 'ForkPoolWorker-7' pid:75952 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 07:15:16,889: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 3.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 3.
"""
[2025-07-16 16:05:12,553: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[924ff6af-4409-4a71-a388-c2df57d2d90e] received
[2025-07-16 16:05:12,569: INFO/ForkPoolWorker-8] Starting processing for filing 70
[2025-07-16 16:05:12,709: ERROR/MainProcess] Process 'ForkPoolWorker-8' pid:75955 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:05:12,724: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 4.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 4.
"""
[2025-07-16 16:07:26,748: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[cdd2e284-407e-4cbd-9b7a-a7ee5c2b0503] received
[2025-07-16 16:07:26,756: INFO/ForkPoolWorker-9] Starting processing for filing 71
[2025-07-16 16:07:26,870: ERROR/MainProcess] Process 'ForkPoolWorker-9' pid:76260 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:07:26,884: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 5.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 5.
"""
[2025-07-16 16:10:50,014: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[da81cdcf-a11e-45b3-afaa-96e5b0d474d3] received
[2025-07-16 16:10:50,021: INFO/ForkPoolWorker-10] Starting processing for filing 72
[2025-07-16 16:10:50,121: ERROR/MainProcess] Process 'ForkPoolWorker-10' pid:76344 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:10:50,135: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 6.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 6.
"""
[2025-07-16 16:29:14,118: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[43353d64-0833-4f59-97b8-de8e3a0999fe] received
[2025-07-16 16:29:14,124: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[e2261701-f4bc-4473-9ac2-c7e6adaceb5d] received
[2025-07-16 16:29:14,126: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[d94a51a6-39cd-4684-82fc-51709c275a51] received
[2025-07-16 16:29:14,126: INFO/ForkPoolWorker-11] Starting processing for filing 73
[2025-07-16 16:29:14,128: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[a0413f9a-b204-4de0-ba17-936a84fd4f5b] received
[2025-07-16 16:29:14,238: ERROR/MainProcess] Process 'ForkPoolWorker-11' pid:76473 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:29:14,252: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 7.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 7.
"""
[2025-07-16 16:29:20,126: INFO/ForkPoolWorker-12] Starting processing for filing 74
[2025-07-16 16:29:20,235: ERROR/MainProcess] Process 'ForkPoolWorker-12' pid:76551 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:29:20,248: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 8.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 8.
"""
[2025-07-16 16:29:26,129: INFO/ForkPoolWorker-13] Starting processing for filing 75
[2025-07-16 16:29:26,265: ERROR/MainProcess] Process 'ForkPoolWorker-13' pid:76557 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:29:26,278: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 9.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 9.
"""
[2025-07-16 16:29:32,130: INFO/ForkPoolWorker-14] Starting processing for filing 76
[2025-07-16 16:29:32,237: ERROR/MainProcess] Process 'ForkPoolWorker-14' pid:76561 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:29:32,250: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 10.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 10.
"""
[2025-07-16 16:37:10,814: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[3ef85865-b4d0-4807-b15c-032b935e14de] received
[2025-07-16 16:37:10,822: INFO/ForkPoolWorker-15] Starting processing for filing 77
[2025-07-16 16:37:10,938: ERROR/MainProcess] Process 'ForkPoolWorker-15' pid:76569 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:37:10,952: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 11.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 11.
"""
[2025-07-16 16:38:18,877: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[7866b1aa-7781-49a1-b960-4913576cd526] received
[2025-07-16 16:38:18,883: INFO/ForkPoolWorker-16] Starting processing for filing 78
[2025-07-16 16:38:18,982: ERROR/MainProcess] Process 'ForkPoolWorker-16' pid:77000 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 16:38:18,996: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 12.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 12.
"""
[2025-07-16 17:02:20,876: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[44d9207e-4bdc-4168-ade4-d5e2d2a73ca8] received
[2025-07-16 17:02:20,888: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[990cf6c8-68f7-474d-be5c-0a69742d481e] received
[2025-07-16 17:02:20,901: INFO/ForkPoolWorker-17] Starting processing for filing 79
[2025-07-16 17:02:20,983: ERROR/MainProcess] Process 'ForkPoolWorker-17' pid:77032 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 17:02:20,997: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 13.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 13.
"""
[2025-07-16 17:02:26,883: INFO/ForkPoolWorker-18] Starting processing for filing 80
[2025-07-16 17:02:26,992: ERROR/MainProcess] Process 'ForkPoolWorker-18' pid:77274 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 17:02:27,006: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 14.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 14.
"""
[2025-07-16 17:33:50,119: INFO/MainProcess] Task app.tasks.filing_tasks.process_filing_task[c0badc91-6108-47cf-a6e9-d243deb021e3] received
[2025-07-16 17:33:50,125: INFO/ForkPoolWorker-19] Starting processing for filing 81
[2025-07-16 17:33:50,228: ERROR/MainProcess] Process 'ForkPoolWorker-19' pid:77278 exited with 'signal 11 (SIGSEGV)'
[2025-07-16 17:33:50,242: ERROR/MainProcess] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV) Job: 15.')
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.einfo.ExceptionWithTraceback: 
"""
Traceback (most recent call last):
  File "/Users/evanhao/Documents/fintellic/venv/lib/python3.11/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
    raise WorkerLostError(
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV) Job: 15.
"""
